cilamp_cfg.py

```diff
@@ -1,5 +1,7 @@
-#!/usr/bin/python
-# coding: utf-8
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+import sys
+import serial
```

Comment:
Breaking PEP8 here was intentional: wanted the python 3 check to be earlier than serial availability check. Any suggestion on how to have both PEP8 and that check order?